### ISSUE TITLE
Web application performance enhancement.

### ISSUE THREAD

**Issue by ParamThakkar123** [AryanNanda17/NonStop-Lofi-Vibe-Generator#48]
When the web application is run on the system, there's some lag sometime possibly due to 3D content and the GLSL shaders involved. We are looking to optimize this load time and increase overall web application performance.

**Comment by nikkuv** [comment]
@ParamThakkar123 Can I try to optimise this ? please assign it to me, I'll inform you if I'm not able to do it after sometime.

**Comment by ParamThakkar123** [comment]
Yeah sure. Thanks for your interest in our project. You can go ahead with your optimizations. assigning this to you

**Comment by AryanNanda17** [comment]
@nikkuv, also it would be really great and motivating for our team if you could star this repo so that this project reaches more people. Thanks again for your interest. I am assigning this to you

**Comment by AryanNanda17** [comment]
@nikkuv, any updates?

**Comment by ParamThakkar123** [comment]
@nikkuv if any more information is needed, we can help you with that. You can use lighthouse to check the site performance.

**Comment by nikkuv** [comment]
@AryanNanda17 @ParamThakkar123 Was trying things to improve performance, it increased just a bit though from 55 to 64 for desktop, But I'm not able to push to the repo. It is saying please ensure if you have the write access to the repo.

**Comment by ParamThakkar123** [comment]
> @AryanNanda17 @ParamThakkar123 Was trying things to improve performance, it increased just a bit though from 55 to 64 for desktop, But I'm not able to push to the repo. It is saying please ensure if you have the write access to the repo.

Did you fork this repo and made changes on the forked repo ? Because this happens when you didn't fork the repo and cloned the upstream and made the changes.

**Comment by AryanNanda17** [comment]
@nikkuv, any updates?

**Comment by AryanNanda17** [comment]
@nikkuv has made some optimizations, but we still need further performance improvements.
